src/formatter.ts

```diff
@@ -556,11 +666,10 @@ export class Formatter {
 
               // Calculate the expected distance of the current context from the last matching tickable. The
               // distance is scaled down by the softmax for the voice.
-              if (options.globalSoftmax) {
+              if (formatterOptions.globalSoftmax) {
                 const t = totalTicks;
-                const exp = (v) => Math.pow(softmaxFactor, v / t);
-                expectedDistance = (exp(maxTicks) / expTicksUsed) * adjustedJustifyWidth;
-              } else {
+                expectedDistance = (softmaxFactor ** maxTicks / t / expTicksUsed) * adjustedJustifyWidth;
```

Comment:
This is wrong too, precedence of `**` is higher than `/`.

Reply:
done